### ISSUE TITLE
Python 3.x support?

### ISSUE THREAD

**Issue by pahaz** [pyjs/original-pyjs#831]
Hello, I didn`t find information about pyhon 3.x support.

What do you think about this?

**Comment by azam58** [comment]
Yes. While I was tryin to install there were many errors, mostly  "invalid syntax"... 
Seems we need to do something... @pahaz

**Comment by ztane** [comment]
well, documentation says that "some of the 2.6 syntax is supported when compiling to javascript" :D

**Comment by wannaphong** [comment]
Now, Python 3.x support?

**Comment by timofonic** [comment]
Any news?

**Comment by briand-no** [comment]
Dang, looks like this project isn't really supported anymore. Or at least there haven't been any updates in a couple years. Would be great to have Python 3 support. Anyone interested in reviving this project?

**Comment by donno2048** [comment]
I know this thread is from literally years ago but I started doing some manual changes #866
